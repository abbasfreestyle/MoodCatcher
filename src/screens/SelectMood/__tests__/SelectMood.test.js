import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { SelectMoodScreen } from '..';

const requiredProps = {
  navigation: { navigate: jest.fn() },
  mood: 6,
  feelings: [],
  onSetMethod: jest.fn()
};

const wrapper = shallow(<SelectMoodScreen {...requiredProps} />);

describe('Screens > Select Mood', () => {
  describe('Snapshots', () => {
    test('should match screen', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
