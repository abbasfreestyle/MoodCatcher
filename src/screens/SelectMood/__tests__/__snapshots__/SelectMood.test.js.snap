// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens > Select Mood Snapshots should match screen 1`] = `
<View
  flex={1}
>
  <Title>
    Select your mood
  </Title>
  <FlatList
    ListHeaderComponent={
      <Column
        alignItems="center"
        flex={1}
        margin={20}
      >
        <View
          marginBottom={-50}
          marginTop={-75}
        >
          <Face
            progress={0.8333333333333334}
            size={350}
          />
        </View>
        <Row>
          <Slider
            animationType="timing"
            debugTouchArea={false}
            maximumTrackTintColor="#A073B2"
            maximumValue={7}
            minimumTrackTintColor="#D4D4D5"
            minimumValue={1}
            onValueChange={[Function]}
            step={1}
            style={
              Object {
                "flex": 1,
              }
            }
            thumbTintColor="#F26AA3"
            thumbTouchSize={
              Object {
                "height": 40,
                "width": 40,
              }
            }
            trackStyle={
              Object {
                "height": 5,
              }
            }
            value={6}
          />
        </Row>
      </Column>
    }
    data={
      Array [
        Object {
          "id": "1",
          "name": "Happy",
        },
        Object {
          "id": "2",
          "name": "Excited",
        },
        Object {
          "id": "3",
          "name": "Anxious",
        },
        Object {
          "id": "4",
          "name": "Nervous",
        },
        Object {
          "id": "5",
          "name": "Relaxed",
        },
      ]
    }
    disableVirtualization={false}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={2}
    onEndReachedThreshold={2}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    updateCellsBatchingPeriod={50}
    windowSize={21}
  />
  <Regular
    disabled={true}
    flex={true}
    margin={10}
    onPress={[Function]}
  >
    NEXT
  </Regular>
</View>
`;
